app/scripts/app.js

```diff
@@ -46,4 +46,5 @@ app.config(function ($routeProvider) {
       redirectTo: '/'
     });
 })
-  .constant('defaultFolder', '2016-01-14');
+  .constant('defaultFolder', '2016-01-14')
```

Comment:
Hi Mateusz it is not, it is one of the items that I fixed before sending pull request